opentelemetry-sdk/src/opentelemetry/sdk/_events/__init__.py

```diff
@@ -49,13 +48,9 @@ def emit(self, event: Event) -> None:
         if isinstance(self._logger, NoOpLogger):
             # Do nothing if SDK is disabled
             return
-        span_context = trace.get_current_span().get_span_context()
```

Comment:
Context is not being passed anymore, but this code is going away so not sure if we want to add it